ash_dal/dao/async_dao.py

```diff
@@ -89,4 +89,10 @@ async def update(self, specification: dict[str, t.Any], update_data: dict[str, t
         async with self.db.session as session:
             result = await session.execute(update(self.__model__).filter_by(**specification), update_data)
             await session.commit()
-            return bool(result.rowcount)   # pyright: ignore
+            return bool(result.rowcount)  # pyright: ignore
+
+    async def delete(self, specification: dict[str, t.Any]) -> bool:
```

Comment:
The same comment as in UPDATE PR, specification can be None in case I want to delete all records.
Also in this case if specification is None then it's better to use Truncate(can not find any sqlalchemy docs about truncate, maybe delete makes it itself but not sure)